content/docs/sidebar.json

```diff
@@ -412,7 +413,10 @@
     "source": "cml/index.md",
     "icon": "cml",
     "children": [
-      "get-started",
+      {
+        "label": "Get Started (GitHub)",
+        "slug": "start-github"
```

Comment:
Took the opportunity to change this URL, which will be needed if we're publishing another Get Started for Gitlab, for example. I just need to check whether a redirect is needed to transfer any search traffic... ⌛

Reply:
Hmm yeah it's already [indexed](https://search.google.com/u/1/search-console/inspect?resource_id=sc-domain%3Advc.org&id=fDlRfKU-LaU5thDLvisO3g)... But it has almost 0 clicks from search yet, so maybe not worth redirecting

![image](https://user-images.githubusercontent.com/1477535/107329837-57ffd000-6a76-11eb-937a-935c3bdd2aad.png)